src/databricks/labs/ucx/runtime.py

```diff
@@ -62,8 +53,8 @@ def all(cls):
             ]
         )
 
-    def tasks(self) -> list[Task]:
```

Comment:
Looks like a property, not a method. And redundant, make the property in the init not hidden